.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,61 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 */3 * * *"
+  workflow_dispatch:
+
+
+jobs:
+  autoupdate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        sudo pip3 install setuptools wheel
+        sudo pip3 install yq
+        pip3 install -r scripts/requirements.txt
+    - name: Run autoupdate
+      run: |
+        git config user.email "actions@github.com"
+        git config user.name "Github Actions"
+        git fetch origin
+        
+        for file in manifests/*
+        do
+          git reset --hard master
+          ./scripts/autoupdate.py "$file"
+          
+          if [[ `git status --porcelain` ]]; then
+            echo "Things have changed, attempting to create a PR"
+            git status
+            
+            version=$(cat "$file" | yq -r ".version") # "1.5.3"
+            filename=${file##*/} # "Plugin Name.yml"
+            name=${filename%.*} # "Plugin Name"
+            branchname=$(echo "$name-$version" | sed 's/[^a-zA-Z0-9]/_/g') # "Plugin_Name-1_5_3"
+            
+            if git branch -al | grep -F "remotes/origin/$branchname" > /dev/null; then
+              echo "Branch $branchname already exists, discarding changes"
+              git restore manifests/
+
+            else
+              git checkout -B "$branchname"
+              git add manifests/
+              git commit -m "Update $name to $version"
+              git push --set-upstream origin "$branchname"
+
+              # the poor man's "does this PR exist?"
+              if ! gh pr ready "$branchname"; then
+                gh pr create --fill --label Autoupdate
+              fi
```

Comment:
We should fill the PR body with a link to the version diff where possible, to make it dead-simple for a maintainer to inspect the changes before merging.

Reply:
Here's the bare minimum: https://github.com/MCOfficer/endless-sky-plugins-1/pull/59